app/cli/cmd/attached_integration_list.go

```diff
@@ -53,22 +53,32 @@ func attachedIntegrationListTableOutput(attachments []*action.AttachedIntegratio
 	fmt.Println("Integrations attached to workflows")
 	t := newTableWriter()
 	t.AppendHeader(table.Row{"ID", "Kind", "Config", "Attached At", "Workflow"})
-	for _, i := range attachments {
-		wf := i.Workflow
-		integration := i.Integration
+	for _, attachment := range attachments {
+		wf := attachment.Workflow
+		integration := attachment.Integration
+
+		// Merge attachment and integration configs
+		// If there are not attachment configuration then create an empty map
+		if attachment.Config == nil {
+			attachment.Config = make(map[string]any)
+		}
+
+		if integration.Config == nil {
+			integration.Config = make(map[string]any)
+		}
 
 		var options []string
-		if i.Config != nil {
-			maps.Copy(i.Config, integration.Config)
-			for k, v := range i.Config {
-				if v == "" {
-					continue
-				}
-				options = append(options, fmt.Sprintf("%s: %v", k, v))
+		maps.Copy(attachment.Config, integration.Config)
```

Comment:
I think this is not correct. We are overwriting the configuration provided at the attachment state, and I think we should either show both (attachment and registration configs) or use the attachment value instead in case the keys are the same.

Reply:
Yeah, that's a good point. I updated it to instead use the attachment config in the case of a conflict.

Thanks!